app/views/communicart_mailer/_comments.html.haml

```diff
@@ -0,0 +1,21 @@
+- if comments.any?
+  %table.submodel-section
+    %tr
+      %td.w-container{:colspan => "2"}
+        %strong Comments
+    - comments.each do |c|
+      %tr
+      - if c.update_comment
+        class="comment-update"
+      \&gt;
```

Comment:
Yeah, nice catch. Looks like a [known issue](https://github.com/haml/html2haml/issues/40) with html2haml which erb2haml uses under the hood.